docker/base

```diff
@@ -200,6 +200,6 @@ LABEL maintainer="Yoan Blanc <yoan@dosimple.ch>" \
       org.opencontainers.image.source="https://github.com/HE-Arc/webapp-server" \
       org.opencontainers.image.schema-version="1.0"
 
-ENTRYPOINT [ "/tini", "--" ]
+ENTRYPOINT [ "bash" ]
```

Comment:
quelle erreur as-tu vu avec tini?

Reply:
Je vois, c'est le `/tini` qui devrait être `/usr/bin/tini` -- évidemment.

Reply:
```suggestion
ENTRYPOINT [ "/usr/bin/tini", "--" ]
```

Reply:
Je reteste aujourd'hui et je te répond ici avec l'erreur exact

Reply:
Hello @greut, désolé pour le délai.
L'erreur était : "ERROR: for base  Cannot start service base: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "/tini": stat /tini: no such file or directory: unknown"
Mais j'ai testé ta solution et oui effectivement mettre le bon chemin fonctionne ^^
Merci de ton aide, on va pouvoir merge cette branch soon.